scripts/test-utils/test-fixture-dir.ts

```diff
@@ -140,7 +144,11 @@ export function testFixtureDir<T>(
             for (const [outputName, content] of outputsList) {
                 const outputPath = path.resolve(dirname, outputName);
                 try {
-                    await expect(content ?? '').toMatchFileSnapshot(outputPath);
+                    if (ssrVersion === 1 && outputName.endsWith('.html')) {
```

Comment:
Would be good to have a brief comment here explaining why `ssrVersion === 1` uses a different matcher.